views/find_missing.py

```diff
@@ -161,9 +161,15 @@ def update_init_search(self):
             print(noteTypesSelected, fieldsSelected)
 
         self.initQuery = ""
+        
         self.initQuery += self.search_formatter(True, "deck", deck_name) if deck_name else ""
+
         self.initQuery += self.search_multiple_terms_formatter("note", noteTypesSelected) if self.model_field_selection_enabled else ""
-        self.initQuery += self.search_multiple_fields_formatter(fieldsSelected, self.REPLACEMENT_STRING) if self.model_field_selection_enabled else ""
+        
+        if self.model_field_selection_enabled:
+            self.initQuery += self.search_multiple_fields_formatter(fieldsSelected, self.REPLACEMENT_STRING) 
+        else:
+            self.initQuery += self.REPLACEMENT_STRING
```

Comment:
I think we need a space before the replacement string to separate it and the deck name if there is one.

Reply:
It already has it :+1: `search_formatter` gives it one by default when the deck name is added, see line 219